### ISSUE TITLE
InvalidArgumentError 

### ISSUE THREAD

**Issue by yashtongia21** [EvilPort2/Sign-Language#4]
InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [47] rhs shape= [45]
	 [[Node: save/Assign_7 = Assign[T=DT_FLOAT, _class=["loc:@logits/bias"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](logits/bias, save/RestoreV2:7)]]

This is the error i am getting while running cnn_tf.py
Kindly provide the solution
